File: src/libs/ThirdParty.CSSuppliers/ExpediaRapid/SerializableClasses/Cancel/CancelResponse.cs
namespace ThirdParty.CSSuppliers.ExpediaRapid.SerializableClasses.Cancel
{

    public class CancelResponse : IExpediaRapidResponse
    {

        public bool IsValid(string responseString, int statusCode)
        {

            switch (statusCode)
            {
                case 202:
                    {
                        return true;
                    }
                case 204:
                    {
                        return true;
                    }
            }

            return false;
        }
    }

}

File: src/libs/ThirdParty.VBSuppliers/Properties/ExpediaRapid/SerializableClasses/Cancel/CancelResponse.vb
Imports Intuitive.Net

Namespace ExpediaRapid.SerializableClasses.Cancel

    Public Class CancelResponse
        Implements IExpediaRapidResponse

        Public Function IsValid(responseString As String, statusCode As Integer) As Boolean Implements IExpediaRapidResponse.IsValid

            Select Case statusCode
                Case 202
                    Return True
                Case 204
                    Return True
            End Select

            Return False
        End Function
    End Class

End Namespace
